src/main/kotlin/no/nav/syfo/behandler/api/BehandlerDTO.kt

```diff
@@ -14,3 +16,18 @@ data class BehandlerDTO(
     val poststed: String?,
     val telefon: String?,
 )
+
+fun List<BehandlerDTO>.removeDuplicates() = this.sortFastlegerFirst().distinctBy { it.behandlerRef }
+
+fun List<BehandlerDTO>.sortFastlegerFirst(): List<BehandlerDTO> {
+    val fastleger = mutableListOf<BehandlerDTO>()
```

Comment:
Går an å bruke https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/partition.html her hvis man har lyst :)
(splitte this i to lister - en med fastleger og en med andre)

Reply:
Bra forslag det. Skal også gå ann å bruke `list.sortedBy()` og så sortere på enum `BehandlerArbeidstakerRelasjonstype`.
Tror først enum-verdi da kommer først, så hvis `FASTLEGE` er først i listen over `BehandlerArbeidstakerRelasjonstype` kommer den først.

Reply:
Vi gjorde faktisk det med sortedBy først, men tenkte dette var mer robust og eksplisitt. At FASTLEGE enum er det høyeste nummeret er jo egentlig litt tilfeldig.

Vi sjekker ut det med partition da, kanskje det blir mer obvious code 😁.